airflow/contrib/operators/dataflow_operator.py

```diff
@@ -335,7 +335,7 @@ class DataFlowPythonOperator(BaseOperator):
         JOB_STATE_RUNNING state.
     :type poll_sleep: int
     """
-    template_fields = ['options', 'dataflow_default_options', 'job_name']
+    template_fields = ['options', 'dataflow_default_options', 'job_name', py_file]
```

Comment:
You need to pass the arguments as strings like so
```suggestion
    template_fields = ['options', 'dataflow_default_options', 'job_name', 'py_file']
```

Reply:
@feluelle don't know how I missed it :)